### PR TITLE
IA-2268: add PT translations for budget tool

### PR DESCRIPTION
**Pull request by codeKameleon**
Budget tool needs to be translated in PT as well

Related JIRA tickets : IA-2268

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- added a pt.json file
- added the missing translations in EN and FR

### CODE REVIEW

**Comment by codeKameleon** [comment]
missing a few translations in EN and FR, will add them tomorrow morning to finish the PR

File: plugins/polio/js/src/constants/translations/en.json [hunk 1]
```diff
@@ -45,6 +45,8 @@
     "iaso.polio.form.label.agNopvGroup": "AG/nOPV Group",
     "iaso.polio.form.label.approval": "Approval",
     "iaso.polio.form.label.approval_confirmed": "Approval confirmed",
+    "iaso.polio.form.label.approve_unicef": "Approve (UNICEF)",
```
Comment (hakifran) [inline plugins/polio/js/src/constants/translations/en.json:48]: This key is not translated in french(fr.json)

File: plugins/polio/js/src/constants/translations/en.json [hunk 2]
```diff
@@ -45,6 +45,8 @@
...
+    "iaso.polio.form.label.approve_who": "Approve (WHO)",
```
Comment (hakifran) [inline plugins/polio/js/src/constants/translations/en.json:49]: Same here(This key is not translated in french(fr.json))

**Review by hakifran — CHANGES_REQUESTED** [review]

File: plugins/polio/js/src/constants/translations/en.json (hunk 2 above)
Comment (quang-le) [reply plugins/polio/js/src/constants/translations/en.json:49]: I did it

**Review by quang-le — APPROVED** [review]

**Review by quang-le — CHANGES_REQUESTED** [review]
Will update with translation reviewed by a native speaker

**Review by hakifran — APPROVED** [review]

**Comment by quang-le** [comment]
Closing as the ticket is back to waiting for client/specification status and will require more work than just adding the JSON

**Pull request closed without merging**